pkg/plugin/shell_plugin.go

```diff
@@ -62,6 +62,11 @@ func (s Sheller) Run(context Context, phase Phase) error {
 				return fmt.Errorf("nodes is not found by label [%s]", on)
 			}
 			allHostIP = ipList
+		} else if on == common.MASTER || on == common.NODE {
+			role := context.Cluster.GetIPSByRole(on)
```

Comment:
So the return values is iplist not role.  role->iplist

Reply:
ok